utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -58,9 +56,19 @@ def clean_build(self, comps=None):
     def run(self):
         return self._run()
 
-    def _run(self, suffix="base"):
+    def _set_active_case(self, case):
```

Comment:
Key change. Allows a system_test to use multiple different cases, but only 1 at a time.